#goal definition
geometry_msgs/Pose targetPose
---
#result definition
---
#feedback definition

